src/js/player.js

```diff
@@ -2473,6 +2477,32 @@ class Player extends Component {
     return this.techGet_('readyState');
   }
 
+  /**
+   * Get a video track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#videotracklist
+   *
+   * @return {VideoTrackList} thes current video track list
+   * @method videoTracks
+   */
+  videoTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but videoTracks should still work.
+    return this.tech_ && this.tech_['videoTracks']();
+  }
+
+  /**
+   * Get an audio track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#audiotracklist
+   *
+   * @return {AudioTrackList} thes current audio track list
+   * @method audioTracks
+   */
+  audioTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but audioTracks should still work.
+    return this.tech_ && this.tech_['audioTracks']();
```

Comment:
If that works, that's fine. Just wanted to give the background on it. Previously we didn't have that property that was used to initialize the tech with.